composer.json

```diff
@@ -35,5 +35,12 @@
         "psr-4": {
             "Swag\\PlatformDemoData\\": "src/"
         }
+    },
+    "scripts": {
+        "cs": "../../../vendor/bin/php-cs-fixer fix --dry-run",
+        "cs-fix": "../../../vendor/bin/php-cs-fixer fix",
```

Comment:
Isn't it `ecs` and `ecs-fix` in platform and plugins?

Reply:
yeah, but due to "backward" compatibility. I would have re-named it everywhere, but people argued, that they have finger memories with "ecs-fix", so we should not break it. 

but here there was nothing before, so I added it with "cs*" (also on other plugins)